src/app/modules/shared/components/month-picker/month-picker.component.ts

```diff
@@ -1,23 +1,68 @@
 import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+import * as moment from 'moment';
```

Comment:
I left a comment before regarding importing everything from libraries. I suggested to see if we need to import all (*) the library or only what we want to use. As I mentioned, in the ui we should avoid importing complete libraries due to performance issues in the future.